### PR TITLE
Add com.termius.Termius

### PR DESCRIPTION
**Pull request by web1n**
Termius is a free SSH and SFTP client for Linux that will make you more productive.

### Please confirm your submission meets all the criteria

- [X] Please describe your application briefly.
- [X] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [X] My pull request follows the instructions at [App Submission][submission].
- [X] I have [built][build] and tested the submission locally.
- [X] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [X] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [ ] I am an author/developer/upstream contributor of the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [X] The domain used for the application ID is controlled by the application developers either directly or through the code hosting (e.g. GitHub, GitLab, SourceForge, etc.). The [application id guidelines][app-id] are followed.
- [X] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*
I added custom fontconfig to fix [Chinese characters displayed as variant (Japanese) glyphs](https://wiki.archlinux.org/title/Localization/Simplified_Chinese#Chinese_characters_displayed_as_variant_(Japanese)_glyphs)

[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[maint]: https://docs.flathub.org/docs/for-app-authors/maintanance
[submission]: https://docs.flathub.org/docs/for-app-authors/submission
[build]: https://docs.flathub.org/docs/for-app-authors/submission/#before-submission
[app-id]: https://docs.flathub.org/docs/for-app-authors/requirements#application-id

### CODE REVIEW

**Comment by web1n** [comment]
bot, build com.termius.Termius

**Comment by hfiguiere** [comment]
> Please describe your application briefly.

this is missing

File: com.termius.Termius.yaml [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '23.08'
+command: termius
+separate-locales: false
+
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
```
Comment (hfiguiere) [inline com.termius.Termius.yaml:13]: either this support wayland out of the box so it's `fallback-x11` or it doesn't and in that case `wayland` should be removed.

Comment (web1n) [reply com.termius.Termius.yaml:13]: It doesn't work out of the box, I will remove wayland.

**Comment by web1n** [comment]
> > Please describe your application briefly.
> 
> this is missing

now added

**Comment by web1n** [comment]
bot, build com.termius.Termius

**Comment by bbhtt** [comment]
Oops, sorry, clicked merge on the wrong thing. Can you please open a new PR?

**Comment by web1n** [comment]
> Oops, sorry, clicked merge on the wrong thing. Can you please open a new PR? @web1n

ok, I will open a new pr

**Pull request merged**